client/image_tag.go

```diff
@@ -11,13 +11,13 @@ import (
 
 // ImageTag tags an image in the docker host
 func (cli *Client) ImageTag(ctx context.Context, source, target string) error {
-	if _, err := distreference.ParseNamed(source); err != nil {
-		return errors.Wrapf(err, "Error parsing reference: %q is not a valid repository/tag", source)
+	if _, err := parseNamed(source); err != nil {
+		return err
 	}
 
-	distributionRef, err := distreference.ParseNamed(target)
+	distributionRef, err := parseNamed(target)
```

Comment:
Ah, right, so yes, we need to have parsing client-side for this. Wondering if we should rethink this endpoint to have it done server-side